brave/browser-laptop
# use ledger-publisher@0.8.65

Test Plan:

1. see if favicons appear more robustly on the "Brave Payments" page

2. set `LEDGER_PUBLISHER_MIN_DURATION=msecs` to define the minimum aggregate number of msec that a publisher must be visited to be visible, voted on, etc

3. set `LEDGER_PUBLISHER_MIN_VISITS=visits` to define the minimum number that a publisher must be visited to be visible, voted on, etc

4. set `LEDGER_PUBLISHER_VISIT_DURATION=msecs` to define the minimum number of msec that a page must be visited to be considered a view.

the defaults for the first two are both are `0` which disables the test. the default for final variables is `8`

enjoy!

/mtr

probably fixes https://github.com/brave/browser-laptop/issues/3684

also introduces two optional environment variables for testing various publisher display strategies

also cleans-up some minor usage of `msecs.minute`

auditor: @diracdeltas
